sepal_ui/sepalwidgets/alert.py

```diff
@@ -18,6 +18,7 @@ class Divider(v.Divider, SepalWidget):
 
     Args:
         class\_ (str, optional): the initial color of the divider
+        kwargs (optional): any parameter from a v.Divider. ['class_'] cannot be overwritten
```

Comment:
why you're using brackets to the kwargs? this can be misunderstood

Reply:
because for some there are multiple one, I'll change it to make it clearer